core/src/App.html

```diff
@@ -741,12 +772,18 @@
           ).then(() => {
             this.set({
               mfModal: {
-                isDisplayed: false
+                displayed: false
               }
             });
           });
         }
       },
+      open(nodepath, settings) {
```

Comment:
Jesus asked me to remove the `SplitView` from there since you always use with the service prefix, eg `SplitViewSvc.open()`. Do you still want to see the change?

Reply:
This would make sense:

```
      openSplitView(nodepath, settings) {
        SplitViewSvc.open(this, nodepath, settings);
      },
      closeSplitView() {
        SplitViewSvc.close(this);
      },
```

What do you think?

Reply:
Oh, its in App.html, of course it should be different there. Thanks